frontend/src/pages/servers/index.tsx

```diff
@@ -195,98 +229,68 @@ const List = ({
           />
-        </span>
-
-        <span className={formItemStyle}>
-          <label className="pr2 self-center" htmlFor="showFull">
-            Full
-          </label>
-          <input
-            type="checkbox"
-            name="showFull"
-            id="showFull"
-            checked={showFull}
-            onChange={(e) => {
-              setShowFull(e.target.checked);
-            }}
-          />
-        </span>
-
-        <span className={formItemStyle}>
-          <label className="pr2 self-center" htmlFor="showEmpty">
-            Empty
-          </label>
-          <input
-            type="checkbox"
-            name="showEmpty"
-            id="showEmpty"
-            checked={showEmpty}
-            onChange={(e) => setShowEmpty(e.target.checked)}
-          />
-        </span>
-
-        <span className={formItemStyle}>
-          <label className="pr2 self-center" htmlFor="sortBy">
-            Sort
-          </label>
-          <select
-            name="sortBy"
-            id="sortBy"
-            value={sort}
-            onChange={(e) => setSort(e.target.value)}
-          >
-            <option value="relevance">Relevance</option>
-            <option value="pc">Players</option>
-          </select>
-        </span>
 
-        {/* <span className={formItemStyle}>
-          <button
-            className={[
-              "ph3",
-              "f6",
-              "pointer",
-              "no-underline",
-              "black",
-              "bg-white",
-              "hover-bg-light-red",
-              "hover-white",
-              "inline-flex",
-              "items-center",
-              "pa2",
-              "ba",
-              "border-box",
-              "mr4",
-            ].join(" ")}
-            type="submit"
+          <Button
+            px="3em"
+            onClick={onOpen}
+            rightIcon={<AddIcon boxSize="0.8em" />}
           >
-            Apply
-          </button>
-        </span> */}
+            Add server
+          </Button>
+          <Modal isOpen={isOpen} onClose={onClose} blockScrollOnMount={false}>
+            <ModalOverlay />
+            <ModalContent top={10}>
+              <ModalHeader>Add a server</ModalHeader>
+              <ModalCloseButton />
+              <ModalBody>
+                <FormControl mb={4}>
+                  <FormLabel>IP or Domain</FormLabel>
+                  <AddServer
+                    onAdd={(server: All) => {
+                      onAdd(server);
+                      onClose();
+                    }}
+                  />
+                  <FormHelperText>
+                    IP must be in format <strong>ip:port</strong>
+                  </FormHelperText>
+                </FormControl>
+              </ModalBody>
+            </ModalContent>
+          </Modal>
+        </Flex>
       </form>
-      <AddServer onAdd={onAdd} />
+      <Stats stats={getStats(data)} />
+
       <ul className="list pl0 mt0 center">
```

Comment:
```suggestion
      <CardList>
```

This will require some modifications to the list items - see above comments